services/php-apache/Dockerfile

```diff
@@ -3,14 +3,15 @@ ENV COMPOSER_NO_INTERACTION=1 \
     COMPOSER_ALLOW_SUPERUSER=1 \
     DOCROOT=/var/www/html
 
-RUN apt-get install -y libfreetype6-dev libjpeg62-turbo-dev libmcrypt-dev libpng-dev && \
+RUN apt-get install -y libfreetype6-dev libjpeg62-turbo-dev libmcrypt-dev libpng-dev libwebp-dev && \
     (apt-get install -y mysql-client || apt-get install -y mariadb-client) && \
     \
     (docker-php-ext-configure gd --with-freetype-dir=/usr/include/ && \
         docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/ || \
         docker-php-ext-configure gd --with-freetype=/usr/include/ --with-jpeg=/usr/include/ || \
         docker-php-ext-configure gd --with-jpeg-dir=/usr/include/ || \
         docker-php-ext-configure gd --with-jpeg=/usr/include/) && \
+        docker-php-ext-configure gd --with-webp && \
```

Comment:
Looks good! I think this line should be outdented one step though, so that it's clearer that it is not included with the previous commands inside of the `(...)`.